server/item/register.go

```diff
@@ -92,6 +92,8 @@ func init() {
 	world.RegisterItem(BlazeRod{})
 	world.RegisterItem(Bone{})
 	world.RegisterItem(Book{})
+	world.RegisterItem(WritableBook{})
+	world.RegisterItem(WrittenBook{})
```

Comment:
Would be nice if these were ordered. If you select all the world.RegisterItem(...) methods, go to `File > Edit > Sort Lines` then it will order them alphabetically